### PR TITLE
Added Cancellation subscription QFC

### PR DESCRIPTION
**Pull request by SeqLaz**
Added section for steps for cancellation subcription

### CODE REVIEW

**Comment by duke-nyuki** [comment]
Task linked: [QF-3015 How to Stop Services in QFieldCloud](https://app.clickup.com/t/2192114/QF-3015)

**Comment by m-kuhn** [comment]
@mbernasocchi does this look good to you?

File: documentation/get-started/tutorials/get-started-qfc.en.md [hunk 1]
```diff
@@ -47,6 +47,14 @@ If the billing information is incomplete, at any time it is possible to finish f
 
 All pricing information is available <a href="https://qfield.cloud/pricing" target="_blank">on the Pricing page</a>.
 
+### Discontinuing Services in QFieldCloud
+
+1. **For Pro Plan Accounts**, click on your username, choose "Settings." **For Organization plan**, click on the name of the Organization (you have to be the owner), click on "Edit Organization".![Enter to setting](../../assets/images/discontinuing_service_01_enter_to_setting.png)
```
Comment (suricactus) [inline documentation/get-started/tutorials/get-started-qfc.en.md:52]:
```suggestion
1. If you need to cancel your subscription on QFieldCloud:
1.1.) For **Pro** plan accounts, click on your username, choose "Settings."
1.2) For **Organization** plan account, click on the name of your Organization, click on "Edit Organization". Note you need to be owner of that Organization.

![Enter to setting](../../assets/images/discontinuing_service_01_enter_to_setting.png)
```

**Review by suricactus — COMMENTED** [review]
General question on formatting, is there a reason why you prefer the images on the same line? It's hard to modify and maintain.

**Comment by SeqLaz** [comment]
> General question on formatting, is there a reason why you prefer the images on the same line? It's hard to modify and maintain.

**Review by mbernasocchi — APPROVED** [review]

**Pull request merged**